src/tools/dev/scripts/bv_support/bv_xdmf.sh

```diff
@@ -166,57 +166,7 @@ diff -c Xdmf/libsrc/XdmfDsmComm.cxx.orig Xdmf/libsrc/XdmfDsmComm.cxx
       }
 EOF
     if [[ $? != 0 ]] ; then
-        warn "Xdmf 2.1.1 Xcode 9 patch failed."
-        return 1
-    fi
-
-    return 0;
-}
-
-function apply_xdmf_gcc_11_2_patch
-{
-    info "Patching Xdmf 2.1.1 for gcc 11.2 . . ."
-    patch -p0 << \EOF
-diff -c Xdmf/libsrc/XdmfDsmComm.cxx.orig Xdmf/libsrc/XdmfDsmComm.cxx
-*** Xdmf/libsrc/XdmfDsmComm.cxx.orig    Fri May 20 12:34:02 2022
---- Xdmf/libsrc/XdmfDsmComm.cxx         Fri May 20 12:34:50 2022
-***************
-*** 50,56 ****
-          XdmfErrorMessage("Cannot Receive Message of Length = " << Msg->Length);
-          return(XDMF_FAIL);
-      }
-!     if(Msg->Data <= 0 ){
-          XdmfErrorMessage("Cannot Receive Message into Data Buffer = " << Msg->Length);
-          return(XDMF_FAIL);
-      }
---- 50,56 ----
-          XdmfErrorMessage("Cannot Receive Message of Length = " << Msg->Length);
-          return(XDMF_FAIL);
-      }
-!     if(Msg->Data == (void*)0 ){
-          XdmfErrorMessage("Cannot Receive Message into Data Buffer = " << Msg->Length);
-          return(XDMF_FAIL);
-      }
-***************
-*** 64,70 ****
-          XdmfErrorMessage("Cannot Send Message of Length = " << Msg->Length);
```

Comment:
was this patch redundant?

Reply:
yes, the xdmf macos patch and the gcc patch were the same. 
I consolidated them to be xdmf_patch_2